k8s/dust-kube-private/apply_infra_dust-kube-private.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -e
+
+function apply_deployment {
```

Comment:
Any chance we can share this between the 2 files ?

Reply:
of course.